### DISCUSSION TITLE
How to implement a chart component, such as a line chart or radar chart?

### DISCUSSION THREAD

**Discussion by Song-aff** [slint-ui/slint#5141] [General]
How to implement a chart component, such as a line chart or radar chart?

**Comment by syf20020816** [comment]
use Path Widget, It allows you use svg command to draw chart

**Comment by zhangzqs** [comment]
how to draw on low memory mcu by software renderer?

**Reply by Vadoola** [reply]
I don't currently have a solution for you, but I know that the Path component is not currently supported on the software renderer. There is a tracking issue for it here: https://github.com/slint-ui/slint/issues/4178
